sql-plugin/src/main/scala/org/apache/spark/sql/rapids/GpuScalaUDF.scala

```diff
@@ -145,7 +145,7 @@ abstract class GpuRowBasedScalaUDFBase(
   /**
    * Spark Scala UDF supports at most 22 parameters.
    */
-  private lazy val wrappedFunc: InternalRow => Any = {
+  private[this] val wrappedFunc: InternalRow => Any = {
```

Comment:
Normally we would want this to be `lazy` since we don't need the driver to evaluate it and it doesn't need to be serialized and sent over to the executors.  Curious what the performance impact is if this remains `lazy`?

Reply:
No, it does not. I have changed it back to be `lazy`. Keeping `lazy` is fine with me.

I just tried to follow how Spark does for this function, so made this diff here.